environment.yml

```diff
@@ -120,6 +120,9 @@ dependencies:
   - tabulate>=0.8.3  # DataFrame.to_markdown
   - natsort  # DataFrame.sort_values
   - pip:
-    - git+https://github.com/pydata/pydata-sphinx-theme.git@master
+      #issue with building environment in conda on windowswith. Issue: https://github.com/pandas-dev/pandas/issues/45123
+      #issue with pydata-sphix-theme on windows. Issue: https://github.com/pydata/pydata-sphinx-theme/issues/523
+      #using this last stable version as workaround
+    - git+https://github.com/pydata/pydata-sphinx-theme.git@41764f5
```

Comment:
can we just use a released pydata-sphinx-theme?

Reply:
Hi @jreback, 

Yes we can. Works fine with the latest stable version of pydata-sphinx-theme. I've changed it accordingly. Thank you for the review. I've removed the previous comments from environment.yml regarding the issue trackers.